OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -129,6 +131,29 @@ public void InvokeCommand(string name, string arg)
 					foreach (var player in world.Selection.Actors.Select(a => a.Owner.PlayerActor).Distinct())
 						world.IssueOrder(new Order("PowerOutage", player, false) { ExtraData = 250 });
 					break;
+
+				case "kill":
+					foreach (var actor in world.Selection.Actors)
+					{
+						if (actor.IsDead)
+							continue;
+
+						if (actor.Info.HasTraitInfo<HealthInfo>())
+							actor.Kill(actor);
+					}
+
+					break;
+
+				case "dispose":
+					foreach (var actor in world.Selection.Actors)
+					{
+						if (actor.IsDead)
+							continue;
```

Comment:
It might occur that someone wants to dispose a dead actor.

Reply:
Thinking about it a bit, can a dead actor be selected? I only added this checks because `levelup` also has it.

Reply:
IIRC a dead actor cannot be selected, but there's probably a condition wherein the actor dies before being deselected. There's likely bitrot in these commands as I wrote most of them a while back.